src/main/java/web/pages/flipkart/PolicySubPage.java

```diff
@@ -0,0 +1,74 @@
+package web.pages.flipkart;
+
+import BasePage.BasePage;
+import com.codeborne.selenide.*;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
+
+public class PolicySubPage extends BasePage {
+    String policyPageHeader = "//*[text()='%s']";
+    public SelenideElement backToTopButton = $x("//div[@id='container']/div[1]/div[4]/div[1]/span");
+    public SelenideElement footer = $(".HJlsB9");
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public boolean isHeaderDisplayed(String policyHeader) {
+
+        logger.info("Welcome to = " + policyHeader);
+        return isDisplayedWait($x(String.format(policyPageHeader, policyHeader)));
+
+    }
+
+    public boolean verifyBackToTopIsDisplayedAndClickButton(String policyElementsWhichNeedToClick) {
+        logger.info("================ String position of scrolling  y axis == " + executeJavaScript("return window.pageYOffset;"));
+
+        if (policyElementsWhichNeedToClick.contains("Sitemap")) {
+            return true;
+        } else {
```

Comment:
Wrong condition applied. Actual Condition should be :
if(isDisplayed(footer))
 return false;
else{
}